### ISSUE TITLE
Rename `course-authoring` -> `authoring`?

### ISSUE THREAD

**Issue by kdmccormick** [overhangio/tutor-mfe#224]
frontend-app-course-authoring was renamed to frontend-app-authoring: https://github.com/openedx/frontend-app-authoring/pull/1289

At a minimum, we should update the URLs here.

Should we rename the app as well? It would be a breaking change for plugin authors.

If so, can we add a `/course-authoring/*` -> `/authoring/*` redirect to caddy to avoid breaking links?

**Comment by regisb** [comment]
Yes, that makes sense, but can I suggest a different url? `/authoring` is not a very conventional path. It's very "English". Should we redirect to `/studio` or `/cms` instead?

I don't have a very strong opinion on this, it's mostly a gut feeling. What do you think?

**Comment by regisb** [comment]
I'm definitely over-thinking this, thanks for indulging me. Let's go with /authoring.

**Comment by arbrandes** [comment]
Fix just got merged.
